fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -137,45 +132,6 @@ public static void start(String starRocksDir, String pidDir, String[] args) {
         }
     }
 
-    // NOTE: To avoid dead lock
-    //      1. never call System.exit in shutdownHook
-    //      2. shutdownHook cannot have lock conflict with the function calling System.exit
-    private static void addShutdownHook() {
-        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
-            LOG.info("start to execute shutdown hook");
-            try {
-                Thread t = new Thread(() -> {
```

Comment:
Why is the addShutdownHook() useless?

Reply:
Shutdown hook is a temporary solution to solve this bug. We have solve it in the bdb code, so it is not needed.